ComputeAdmin/AzureStack.ComputeAdmin.psm1

```diff
@@ -150,7 +150,14 @@ Function Add-VMImage{
         }
     }
 
-    $uri = $armEndpoint + '/subscriptions/' + $subscription + '/providers/Microsoft.Compute.Admin/locations/' + $location + '/artifactTypes/platformImage/publishers/' + $publisher
+    if ($ArmEndpoint.EndsWith("/")) 
```

Comment:
Why can't we just use the TrimEnd function here instead of a large if statement?